PracticePrograms/src/EighthJune/ClockwiseRotationOfArray.java

```diff
@@ -0,0 +1,27 @@
+package EighthJune;
+
+import java.util.Arrays;
+
+public class ClockwiseRotationOfArray {
+
+    public static void leftRotate(int arrayToBeRotated[], int numberOfPositionsToRotate, int SizeOfTheArray) {
+        for (int index = 0; index < numberOfPositionsToRotate; index++)
+            leftRotatebyOne(arrayToBeRotated, SizeOfTheArray);
+    }
+    public static void leftRotatebyOne(int arrayToBeRotated[], int SizeOfTheArray) {
+        int index, temporary;
```

Comment:
merge this two method leftRotate and leftRotatebyOne

Reply:
Done